### DISCUSSION TITLE
Welcome to Mutation++ Discussions!

### DISCUSSION THREAD

**Discussion by jbscoggi** [mutationpp/Mutationpp#150] [General]
## 👋 Welcome All!
  We’re using Discussions as a place to connect users and developers of Mutation++. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us how you use Mutation++.

**Comment by RaminShilav** [comment]
Ramin Shilav,  R&D Engineer, conducting research concerning ablative materials in solid rocket motors, developed PARC - ablative response code, dealing also with instrumentation development. I use mutationpp for simulating boundary layer properties  and generating B' tables.
